subsys/shell/backends/Kconfig.backends

```diff
@@ -358,11 +358,22 @@ config SHELL_BACKEND_TELNET
 	bool "TELNET backend."
 	depends on NET_TCP
 	depends on NET_IPV4 || NET_IPV6
+	select NET_SOCKETS_SERVICE
+	select NET_SOCKETS
 	help
 	  Enable TELNET backend.
 
 if SHELL_BACKEND_TELNET
 
+config SHELL_TELNET_INIT_PRIORITY
+	int "Initialization priority"
+	default 95
+	range 0 99
```

Comment:
Perhaps we could put a range here that takes the service init priority into account, would something like this work (not tested)
```
range NET_SOCKETS_SERVICE_INIT_PRIO 99
```
And as the range does not allow arithmetic, the already existing build assert verifies that the init values are not the same.

Reply:
Ok, it seems that it worked, so updated the range. It's a bit pity we can't do a simple +1 there.